code/iaas/ha/src/main/java/io/cattle/platform/ha/monitor/impl/PingInstancesMonitorImpl.java

```diff
@@ -1,7 +1,17 @@
 package io.cattle.platform.ha.monitor.impl;
 
+import static io.cattle.platform.core.constants.ContainerEventConstants.*;
```

Comment:
So much changed in this class, you might be better off _not_ looking at the diff and just looking at the updated version on its own.